proof/refine/ARM_HYP/TcbAcc_R.thy

```diff
@@ -1786,6 +1787,7 @@ lemma setQueue_nosch[wp]:
 lemma gq_wp[wp]: "\<lbrace>\<lambda>s. Q (ksReadyQueues s (d, p)) s\<rbrace> getQueue d p \<lbrace>Q\<rbrace>"
   by (simp add: getQueue_def, wp)
 
+(* see getObject_TCB_corres and corres_get_tcb *)
```

Comment:
lemma getObject_TCB_corres:
  "corres tcb_relation (tcb_at t) (tcb_at' t)
          (gets_the (get_tcb t)) (getObject t)"

lemma get_tcb_corres:
  "corres tcb_relation (tcb_at t) (tcb_at' t) (gets_the (get_tcb t)) (getObject t)"

lemma corres_get_tcb:
  "corres (tcb_relation \<circ> the) (tcb_at t) (tcb_at' t) (gets (get_tcb t)) (getObject t)"